src/FffGcodeWriter.cpp

```diff
@@ -1727,25 +1727,25 @@ bool FffGcodeWriter::processSingleLayerInfill(const SliceDataStorage& storage, L
                            /*bool skip_some_zags =*/ false,
                            /*int zag_skip_count =*/ 0,
                            mesh.settings.get<coord_t>("cross_infill_pocket_size"));
-        infill_comp.generate(infill_polygons, infill_lines, mesh.cross_fill_provider, &mesh);
-        if (zig_zaggify_infill)
+        infill_comp.generate(infill_polygons, infill_lines_here, mesh.cross_fill_provider, &mesh);
+        if (density_idx < last_idx)
         {
-            if (density_idx == last_idx)
+            auto get_cut_offset = [](const bool zig_zaggify, const coord_t line_width, const size_t line_count)
```

Comment:
True, this was a remnant of an older more complex function.
But changing that will also create a code duplication since the this cutting is done in exactly the same manner in two locations

Reply:
It's fine, just a remark. Just wondering why you did it this way.